### DISCUSSION TITLE
进到后室了!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by whatisyoupassword** [share121/inter-knot#2461] [其他]
像往常一样进了巴莱大厦附件空洞,没想到是后室!!!吓得萝卜都丢了,救命,
!!!!!!!!!SOS!!!!!!!!!!!
![20221110105934_12424](https://github.com/user-attachments/assets/13d17753-5e13-4941-be71-6835f69959ff)

**Comment by haily12345** [comment]
这个委托我......接不了，祝好运

**Comment by GanlinSama** [comment]
几天前也有个哥们掉到这里了，我和好几个专业绳匠一起去捞都没捞起来，连人都没见着，现在也不知道咋样了……
总之楼主自求多福吧，趁着还有网络就别刷绳网了，多陪陪父母吧<img src="https://bbs-static.miyoushe.com/static/2023/05/22/a2fa805a0a782cdb8ab44ddaf8d5343d_9213146029750742963.png" alt="阿姬倒地了02" width="75" >

**Comment by qiuledadada** [comment]
祝楼主好运，前几天也有个兄弟进去了你们大概能遇见

**Comment by springope** [comment]
插眼，等后续

**Comment by UnquestionableFairy** [comment]
这个委托我接了！萝卜不好用，我带了列车长，会出去的<img src="https://img-static.mihoyo.com/communityweb/upload/5f05cd9b0bad4ad7e3bc979fc8b4ab78.png" alt="帕姆 喝茶" width="75" >

**Comment by MikUranium** [comment]
多收集一些杏仁水

**Comment by zhixuan45** [comment]
多走走看看有没有红色的门，门里面应该有东西，但是千万！！千万！！不要靠近写有"happy"的地方

**Comment by OttoApocalypse01** [comment]
这个委托我不接

**Comment by 13520rygvv** [comment]
蹲一手后续

**Comment by TetsuyaTetsuya** [comment]
好恐怖，楼主还活着吗？小道消息，快联系六分街的录像店兄妹，我之前亲眼看到维多利亚家政的人进去了，好像还跟白祇重工有关系，没准他们真有办法救你出去！

**Comment by I-m-Miss-Lucy-s-dog** [comment]
兄弟 我也掉到那了 我是从莱姆尼安空洞掉进去的 等等看能不能碰面

**Comment by feathink0818** [comment]
不是……你等等我试试能不能从量子之海进去
一定要撑住啊喂！

**Comment by JinLingxi** [comment]
兄弟我也去过，好不容易跑出来，祝好运！！！

**Comment by LuAn-afl** [comment]
太好了是实体我们死定了<img src="https://bbs-static.miyoushe.com/static/2023/11/07/5f0581151a00ca4efe963928c2607df0_5889753033090282341.png" alt="尾巴 就这" width="75" >
